docs/malware-scanner/README.md

```diff
@@ -1 +1,44 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+# Malware scanner and detector
+
+---
+
+# Dynamic analysis 
+A slow but reliable detector looking for jar capabilities, potential malware sign
+- run static analysis to filter known threats
+- downloading stuff from the internet (potentially jars, but can be genuine)
```

Comment:
Phrasing: We should be _detecting the capability_ to download, but not actually execute any logic.